MaskRCNN/train.py

```diff
@@ -309,8 +412,9 @@ def check_and_log(cmd):
             starting_epoch=cfg.TRAIN.STARTING_EPOCH
         )
 
-
-        if is_horovod:
+        if args.async_eval:
+            trainer = AsyncTrainer(average=True)
```

Comment:
> This is a nit. Maybe it would be clearer to call this 'AsyncHorovodTrainer'?

Good point. I changed it. Check my [new commit](https://github.com/aws-samples/mask-rcnn-tensorflow/pull/11/commits/77261306d7ce332fb838abf68d1e21c1a4495590)